### DISCUSSION TITLE
Test packaging

### DISCUSSION THREAD

**Discussion by dmthuc** [ByConity/ByConity#110] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/ByConity/ByConity/releases/tag/'>Test packaging</a>.</em>

**Comment by hustnn** [comment]
https://github.com/ByConity/ByConity/releases/tag/0.1.1.1-testing

**Comment by WillemJiang** [comment]
Please add the change log into the release annouce, it can help user to find out what kind of bug fix is done.

**Reply by dmthuc** [reply]
yeah, the new release have change log
